tools/e2e-commons/config/playwright.config.default.mjs

```diff
@@ -55,8 +60,19 @@ const playwrightConfig = {
 		storageState: config.get( 'temp.storage' ),
 		userAgent:
 			'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36 wp-e2e-tests',
+		locale: 'en-US',
+		contextOptions: {
+			reducedMotion: 'reduce',
+			strictSelectors: true,
```

Comment:
We can comment `strictSelectors` out to let the existing ambiguous test selectors pass.